### PR TITLE
Add encypted secrets

### PR DESCRIPTION
**Pull request by DenysHalkin**

### CODE REVIEW

File: .env.dev.aws.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+VITE_API_BASE_URL: ENC[AES256_GCM,data:yzAevR3tTEfK/ETekvhgPm/9ws6ovFOv9c1K2OvWmKOPs9+EzxQaCGUIXs8PoZqnHu23UOAOFA==,iv:WQkcddTt23MGoFMT1XYKY9NyVgraB6iEJA9UAXAkkck=,tag:2SPwA3NjKHmYVRrlR44dgA==,type:str]
```
Comment (vnko1) [inline .env.dev.aws.yaml:1]: additionally, I also use the following values: VITE_BASE_OAUTH_STATE=9lu8d7a9; 
VITE_STATE_ID_COOKIE_LIFETIME=86400

File: .env.dev.aws.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+VITE_GOOGLE_BASE_URL: ENC[AES256_GCM,data:OIFSX6AqfUWjIb9IMZAUiWuM/nBvStq9i5DHiRLxC4ZzFWh4,iv:ZfPn8CHn4yq1Z68g9XdZjsPEJYIORQaAId0ERB7zhJs=,tag:MQ9Yd0nGGpOr8olM8UlQtA==,type:str]
+VITE_GOOGLE_CLIENT_ID: ENC[AES256_GCM,data:7/sNQUc58k8qFiuVApRu81ipA/7Fq8vqKz2YbRFdyv3+tWAF3ej81nRGFwOn4LH1nPmFcOqVvoXlXYVSporPDyeBk6xFBYZj,iv:E6AooSsRb220mwLA6w8rFLl+PfSJdnU1kWBfRU4hd14=,tag:T4oFxFo5lQHhTjp9ibZ0qA==,type:str]
+VITE_GOOGLE_CLIENT_SECRET: ENC[AES256_GCM,data:AOLADVCAxZN4oaG3Vhgwf+HMRaUskoCtV/OW4rRUircAHNw=,iv:Kq9fpcKAII8xjWHXm6SlF6WglOx/rabJqc7LQd7Cfmc=,tag:8S/L/FGx6geuha5sJ2FtTg==,type:str]
+VITE_GOOGLE_CLIENT_STATE: ENC[AES256_GCM,data:MyrYzqlqZMt15Ai8UWYCcvBeUQhEFsME7LgEx1CQfjA8KDrZpurcAR4A,iv:b2DKFb5CHMK4kr9tKLbR9RX05RjPHbyiBTKYBkVdPyY=,tag:u2eDYM+JeiylhZMuc1/1Ag==,type:str]
```
Comment (vnko1) [inline .env.dev.aws.yaml:5]: VITE_GOOGLE_STATE

**Review by vnko1 — CHANGES_REQUESTED** [review]

File: .env.dev.aws.yaml (hunk 1 above)
Comment (DenysHalkin) [reply .env.dev.aws.yaml:1]: Added

File: .env.dev.aws.yaml (hunk 2 above)
Comment (DenysHalkin) [reply .env.dev.aws.yaml]: Changed

**Review by vnko1 — APPROVED** [review]

**Review by ArtemBohak — APPROVED** [review]

**Pull request merged**